### PR TITLE
fix(popup): clear active interception when clear recorded data

### LINKED ISSUE #149: After clear : the interception is still active
### Steps to reproduce
1. Record or add rule
2. Update the response and intercept
3. make the request and receive intercepted response
4. Clear the requests
5. make the request again and see the webpage receive intercepted response

### Expected
After clearing the requests the interception should not be active

### Actual
After clearing the requests the interception is still active

### PR DESCRIPTION
**Pull request by revathskumar**
Fix #149

### CODE REVIEW

**Comment by amit-mb** [comment]
Yes, this works without having to call `server.restore()`.

In addition to this. We need to clear the store values as well.

Attaching a screenshot of store values after clearing the store: 
screen shot 2018-08-21 at 2 23 58 pm

We need to clear `requests` and `requestRecords`. Shall I take this up? @revathskumar

File: app/message_service.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@ export function disableLogging(tabId: number) {
 
 export function clearData(tabId: number) {
   chrome.runtime.sendMessage({ message: "CLEAR_DATA", tabId });
```
Comment (revathskumar) [inline app/message_service.ts]: @amit-mb Is message is suppose to clear the data from redux store right?

**Comment by revathskumar** [comment]
@amit-mb Please test it again.

File: app/reducers/rootReducer.ts [hunk 2]
```diff
@@ -126,7 +126,8 @@ export const reducer = (state = INITIAL_POPUP_STATE, action: Action) => {
       });
     case actionType.CLEAR_REQUESTS:
       return extendTabRecords(state, action.payload, {
-        requests: []
+        requests: [],
+        requestRecords: {}
```
Comment (revathskumar) [inline app/reducers/rootReducer.ts]: @amit-mb  I think now the requestRecords will get cleared from the redux store.

Comment (amit-mb) [reply app/reducers/rootReducer.ts]: Yes, this is what I was referring to 😄

**Pull request merged**